components/inputs/test/input-helper.js

```diff
@@ -65,9 +65,9 @@ export function getRect(page, selector) {
 	});
 }
 
-export function getRectTooltip(page, selector, tooltipIndex) {
-	return page.$eval(selector, (elem, tooltipIndex) => {
-		const content = elem.shadowRoot.querySelectorAll('d2l-tooltip')[tooltipIndex ? tooltipIndex : 0];
+export function getRectTooltip(page, selector) {
```

Comment:
This more generic version replaces the other two versions that are being removed.